core/src/stmt.rs

```diff
@@ -9,3 +10,9 @@ pub trait MongoStatement {
     // Fails if the first row has not been retrieved (next must be called at least once before getValue).
     fn get_value(&self, col_index: u16) -> Result<Option<&Bson>>;
 }
+
+impl Debug for dyn MongoStatement {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        write!(f, "MongoStatement")
```

Comment:
I was getting an error that `Debug` for `dyn MongoStatement` wasn't implemented so I added this.  This area I could use some guidance as well of what to have here.

Reply:
@rychipman , @pmeredit Any comments or suggestions here?
All I could find is it might be easier to just make Debug a super trait of MongoStatement so that there is no need for implementing Debug here. But I am not too sure if this is a good solution.

Reply:
I think making `Debug` a super trait is a fine idea. It will also result in better debug output for `MongoStatements`.

Reply:
Sounds good.  I made `Debug` a super trait for `MongoStatement` and that automatically implemented the Debug trait 👍.  Thanks for the help!